libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -727,6 +730,26 @@ export class DiscoveryProvider {
     return await this._makeRequest(req)
   }
 
+  async getTrendingPlaylists(
```

Comment:
since there are a lot of optional params, does it make sense to use object? or do we want to have same api as the request function?